arch/ARM/STM32/driver_demos/demo_gpio_direct_leds/demo_gpio.gpr

```diff
@@ -0,0 +1,53 @@
+with "../../../../../boards/common_config.gpr";
```

Comment:
This test should be added `the scripts/build_all_examples.py` to make sure we don't break it in the future.